Applications/spire/source/charting/charting_technicals_panel.cpp

```diff
@@ -185,12 +197,12 @@ void ChartingTechnicalsPanel::update_change_label() {
     auto change_text = QString(" " + m_item_delegate->displayText(
       QVariant::fromValue(change), QLocale()) + " (" +
       m_item_delegate->displayText(QVariant::fromValue(
-        (change / *(m_model->get_open())) * (100 * Money::ONE)),
+        100 * (change / *(m_model->get_open()))),
         QLocale()) + "%)");
-    if(change > Money()) {
+    if(change > Money::ZERO) {
       change_text = change_text.replace(0, 1, "+");
       color = QColor("#1FD37A");
-    } else if(change < Money()) {
+    } else if(change < Money::ZERO) {
```

Comment:
What happens if the change is 0?

Reply:
The the change label text format is XX.XX without a +/- sign and the text color is white.